plugins/fluentd_telemetry_plugin/src/merge_configuration_files.py

```diff
@@ -0,0 +1,68 @@
+"""
```

Comment:
I would suggest to move this utility file to `ufm_sdk_tools` repository, it will be shared between all plugins

Reply:
done